src/runtime/components/server/runtime_support_server.cpp

```diff
@@ -272,7 +272,7 @@ namespace hpx { namespace components { namespace server
 {
     ///////////////////////////////////////////////////////////////////////////
     runtime_support::runtime_support(hpx::util::runtime_configuration & cfg)
-      : stop_called_(false), stop_done_(false), terminated_(false),
+      : stop_called_(true), stop_done_(true), terminated_(false),
         dijkstra_color_(false), shutdown_all_invoked_(false),
```

Comment:
@msimberg: Is that change still necessary?

Reply:
Good catch. In principle yes, since the change is independent from the networking changes. It just happened to be triggered once these changes were introduced. I'll check and open a new PR with that change only if needed.